src/webui/www/private/preferences_content.html

```diff
@@ -470,27 +771,45 @@
     </fieldset>
 
     <fieldset class="settings">
-        <legend><input type="checkbox" id="use_dyndns_checkbox" onclick="updateDynDnsSettings();" />
-            <label for="use_dyndns_checkbox">QBT_TR(Update my dynamic domain name)QBT_TR[CONTEXT=OptionsDialog]</label></legend>
+        <legend>
+            <input type="checkbox" id="use_dyndns_checkbox" onclick="updateDynDnsSettings();" />
+            <label for="use_dyndns_checkbox">QBT_TR(Update my dynamic domain name)QBT_TR[CONTEXT=OptionsDialog]</label>
+        </legend>
         <select id="dyndns_select">
             <option value="0">DynDNS</option>
             <option value="1">NO-IP</option>
         </select>
-        <input type="button" value="QBT_TR(Register)QBT_TR[CONTEXT=OptionsDialog]" onclick="registerDynDns();" /><br/><br/>
-        <div class="formRow">
-            <label for="dyndns_domain_text" class="leftLabelMedium">QBT_TR(Domain name:)QBT_TR[CONTEXT=OptionsDialog]</label><input type="text" id="dyndns_domain_text" />
-        </div>
-        <div class="formRow">
-            <label for="dyndns_username_text" class="leftLabelMedium">QBT_TR(Username:)QBT_TR[CONTEXT=OptionsDialog]</label><input type="text" id="dyndns_username_text" />
-        </div>
-        <div class="formRow">
-            <label for="dyndns_password_text" class="leftLabelMedium">QBT_TR(Password:)QBT_TR[CONTEXT=OptionsDialog]</label><input type="password" id="dyndns_password_text" />
-        </div>
+        <input type="button" value="QBT_TR(Register)QBT_TR[CONTEXT=OptionsDialog]" onclick="registerDynDns();" />
+        <table style="margin-top: 10px;">
+            <tr>
+                <td>
+                    <label for="dyndns_domain_text">QBT_TR(Domain name:)QBT_TR[CONTEXT=OptionsDialog]</label>
+                </td>
+                <td>
+                    <input type="text" id="dyndns_domain_text" />
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <label for="dyndns_username_text">QBT_TR(Username:)QBT_TR[CONTEXT=OptionsDialog]</label>
+                </td>
+                <td>
+                    <input type="text" id="dyndns_username_text" />
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <label for="dyndns_password_text">QBT_TR(Password:)QBT_TR[CONTEXT=OptionsDialog]</label>
+                </td>
+                <td>
+                    <input type="password" id="dyndns_password_text" />
+                </td>
+            </tr>
+        </table>
     </fieldset>
 </div>
 
-<br/>
-<div style="text-align: center;"><input type="button" value="QBT_TR(Save)QBT_TR[CONTEXT=HttpServer]" onclick="applyPreferences();" /></div>
+<div style="text-align: center; margin-top: 15px;"><input type="button" value="QBT_TR(Save)QBT_TR[CONTEXT=HttpServer]" onclick="applyPreferences();" /></div>
```

Comment:
can you use em instead of px?

Reply:
What do we gain from using `em` for a margin? The margin isn't calculated in terms of the font-size.

Reply:
>The margin isn't calculated in terms of the font-size.

If I understand correctly, the `<br/>` and `margin-top: 15px;` are for aesthetic reasons, then I think the layout (and hence the margin) should be related to font size and not being an absolute size.

Reply:
I'm still not sure I agree with this but don't feel strongly so will change it.